### DISCUSSION TITLE
Raffle Contract: ( error: a value is required for '--fork-url <URL>' but none was supplied)

### DISCUSSION THREAD

**Discussion by Muhdhammad** [Cyfrin/foundry-full-course-cu#2287] [General]
Hey everyone, 
I'm getting this error when running make deploy-sepolia:
error: a value is required for '--fork-url <URL>' but none was supplied
```
deploy-sepolia:
	@forge script scripts/DeployRaffle.s.sol:DeployRaffle --rpc-url $(SEPOLIA_RPC_URL) --account key --broadcast --verify --etherscan-api-key $(ETHERSCAN_API_KEY) -vvvv
```

I have tried to do source .env but still no fix. Interestingly, when I run forge test --fork-url $SEPOLIA_RPC_URL it works fine.

**Comment by EngrPips** [comment]
Hello @Muhdhammad, run `foundryup`, then `forge clean`, and then retry. Let's see if the problem persists. Please make sure to `source .env` before running the `deploy-sepolia` command, and also make sure your `.env` file is in the root directory of your workspace.

**Reply by Muhdhammad** [reply]
Hey, still no good
Tried ```foundryup``` also ```forge clean```, ```.env``` is in my root directory checked it by running ```ls -a```. Also tried to run script without makefile still it can't recognize the ```$SEPOLIA_RPC_URL``` and ```$ETHERSCAN_API_KEY```, but if I do ```echo $SEPOLIA_RPC_URL``` everything works fine.
 
![error](https://github.com/user-attachments/assets/8469e179-665e-46b5-915f-7447ab94a728)

**Reply by Muhdhammad** [reply]
Oh man, how dumb I am.
I forgot to add this line ```-include .env``` in my makefile. Also it was ```forge script script``` and not ```forge script scripts```. It is working perfectly now.
@EngrPips Thanks for your time!

**Reply by EngrPips** [reply]
You are welcome, Friend. Keep crushing it.
